inference-engine/tests/ngraph_functions/src/gather_nd.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include <memory>
+
+#include "ngraph_functions/builders.hpp"
+
+namespace ngraph {
+namespace builder {
+
+std::shared_ptr<Node> makeGatherND(
+        const ngraph::Output<Node>& dataNode,
```

Comment:
fix code alignment please
arguments fuse with function body

Reply:
Do we such requirements in code style? In my opinion it is more readable.